### ISSUE TITLE
skip_ansible_lint does not work with this action.

### ISSUE THREAD

**Issue by bbaassssiiee** [tsukinowasha/ansible-lint-rules#5]
```
- name: 'configure postgres_net to postgres_port in firewalld'
  become: yes
  firewalld: rich_rule='rule family="ipv4" source address="{{ postgres_net }}" port port="{{ postgres_port }}" protocol="tcp" accept' permanent=true state=enabled immediate=yes
  tags:
    - skip_ansible_lint
```

**Comment by shirou** [comment]
This is an ansible-lint issue. Since `line too long` checks every lines, `tag` can not be used with current ansible-lint implementation.

For example, the default rule `[ANSIBLE0002] Trailing whitespace` is detected though with skip_ansible_lint tag.

**Comment by bbaassssiiee** [comment]
Has this issue been filed with ansible-lint?

**Comment by shirou** [comment]
I have not searched yet. But I think it looks like a little bit hard to fix.
